### DISCUSSION TITLE
Request to change the chime that plays before a TTS announcement.

### DISCUSSION THREAD

**Discussion by veitw** [music-assistant/support#896] [Feature requests and ideas]
I'd like to change the chime sound that plays when I send TTS notifications to `media_player.mass_*` devices.
Is there a pointer I can set to an alternative media file, or a file to replace?

Thanks!

**Comment by OzGav** [comment]
What chime sound? What device type are you sending it to? If it is a Google speaker then that is a setting in the Google Home app.

**Reply by veitw** [reply]
My devices are custom-built ESP32-based speakers running ESPhome utilising the media_player and api components for HA integration.

A chime is played before a TTS notification starts playing. This only happens when sending to `media_player.mass_*` entities, but not using the original `media_player` entities, so my assumption was that this chime sound is injected by Music Assistant.

**Comment by marcelveldt** [comment]
He means the bell sound to announce a TTS message.

For now, this sound is built-in and can not be replaced. Please make this a feature request and if there's enough interest we can add a configurable announce sound. An easy first step in between could be a small toggle in the player settings to disable the announce bell.

**Reply by OzGav** [reply]
Title and post category changed

**Reply by veitw** [reply]
Thank you!

**Reply by addrum** [reply]
Can I double check this discussion counts as a feature request (can see it's in that category)? Or does it need to be turned into an issue?

**Reply by OzGav** [reply]
It’s not an issue as it is working as designed.

**Comment by rjing001** [comment]
Yes Please GET RID of that Annoying BELL!!!!!

**Reply by marcelveldt** [reply]
> Yes Please GET RID of that Annoying BELL!!!!!

Please get rid of that attitude!

**Reply by rjing001** [reply]
Yes ANNOYING isn't it, just like that Bell.

**Reply by rjing001** [reply]
Doesn't matter for me anymore, too many issues.  I uninstalled it altogether.  Maybe I'll try it again later if it get more mature.

**Comment by tiaanv** [comment]
https://github.com/orgs/music-assistant/discussions/2187
There is a way if the devs feel it it worth including..

**Comment by pgale** [comment]
I'd LOVE to see a customisable pre-announce. I use a combination of Sonos and snapcast clients to play music and announce. The bell isn't my favourite sound. I believe the Sonos speakers are limited in that sound? When it comes to the snapcast devices, that must be controlled by HA or MA isn't it? Can that bell be changed? Thanks.

**Comment by daifel** [comment]
I would also be happy if it would be possible to change the pre-announce audio. I am using the docker version of the music assistant server and as a workaround I mount my custom audio file with: 
`-v /path/to/custom/audio/custom-pre-announce.mp3:/app/venv/lib/python3.14/site-packages/music_assistant/helpers/resources/announce.mp3:ro`

Of course, this only works as long as the path in the docker image does not change, e.g. due to a new python version.

**Reply by janstadt** [reply]
Does this still work? im browsing to that directory and i dont have a music_assistant path in site-packages running the latest beta in docker.

**Reply by janstadt** [reply]
Looks as though its moved here for anyone wondering: `/app/venv/lib/python3.12/site-packages/music_assistant/helpers/resources/announce.mp3`

**Reply by daifel** [reply]
Yes, the path has changed. I updated my post above.

**Reply by thematrixdev** [reply]
I am glad that you have shared the path. Thank you very much.

**Comment by peetipablo** [comment]
I'd also love the ability to customize the pre-announce sound! This would be great.

**Comment by Davis8483** [comment]
YES, we need this!!!

**Comment by MarkHofmann11** [comment]
Long time HA user here and somehow wasn't aware of Music Assistant until a week ago.  A fantastic replacement for the default media_player with so many more options.  I finally have annoucements with resume audio on my Google mini's without using an automation hack.

I read this thread and agree 100%, adding custom pre-annoucement sounds would be a great enhancement!  For those that don't want to hear anything, they could use a 1sec-silent.mp3 or something to that effect.  Or you could create whatever you like.  Hope to see that enhancement one day. 

Thanks for all your work on this excellent project!

**Reply by OzGav** [reply]
You can already turn off the pre announcement sound in the settimgs

**Reply by MarkHofmann11** [reply]
> You can already turn off the pre announcement sound in the settimgs

I have the pre-announcements turned off in both the automation (use_pre_announce: false) and disabled in the settings on each Google Mini speaker and I still hear the default Google ding before the annoucement.  

I was hoping to be able to either completely disable that Google ding or replace it with a silent pre-announcement.  Maybe this is unique to the Google Minis, but I still hear it on both the Minis and Google Home hub.

**Reply by marcelveldt** [reply]
The Google ding is a google "ding" - MA has no control over that.

**Comment by wjb178** [comment]
This feature would be nice. I would like to be able to set my speaker pre announce tone to match the alexa echo dot tone.

**Comment by Malvazar** [comment]
The ability to change the pre announcement sound would be lovely, the little bell chime is a bit too much for my ears.

**Comment by OzGav** [comment]
This feature will be available in the next beta.

**Comment by robin-thoni** [comment]
And there it is in `2.7.0`!

<img width="516" height="534" alt="Screenshot from 2025-12-18 12-19-16" src="https://github.com/user-attachments/assets/678c23b9-5cae-4642-b116-305263efde2b" />

I would have loved to be able to specify a URL when calling `music_assistant.play_announcement` so we could select a different one for different use-cases (random info vs urgent stuff, randomly choose among multiple ones, etc). But that's already really cool!

**Reply by OzGav** [reply]
That is an oversight and will be added soon

**Reply by robin-thoni** [reply]
Awesome! Thanks for your hard work guys!

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
